apps/dialtone-documentation/docs/components/stack.md

```diff
@@ -60,19 +71,42 @@ Row: flow horizontally
     </div>
 </code-well-header>
 
-```html
-<div class="d-stack d-stack--row d-stack--sm--column d-stack--lg--column-reverse d-stack--gap-100">
-  <div class="d-bgc-magenta-100">
-    Stack item 1
-  </div>
-  <div class="d-bgc-magenta-100">
-    Stack item 2
-  </div>
-  <div class="d-bgc-magenta-100">
-    Stack item 3
-  </div>
+<code-example-tabs
+htmlCode='
+<div class="d-stack d-stack--row d-stack--gap-500">
+  <div class="d-bgc-magenta-100">Stack item 1</div>
+  <div class="d-bgc-magenta-100">Stack item 2</div>
+  <div class="d-bgc-magenta-100">Stack item 3</div>
 </div>
-```
+'
+vueCode='
+<dt-stack
+  gap="500"
+  direction="row"
+>
+<span class="d-badge">Co-host</span>
+<span class="d-badge">Customer</span>
+<span class="d-badge">
+  <span class="d-badge__icon-left">
+    <dt-icon
+      name="lock"
+      size="200"
+    />
+  </span>
+  <span class="d-badge__label">Locked</span>
+</span>
+<span class="d-badge">
+  <span class="d-badge__icon-left">
+    <dt-icon
+      name="message"
+      size="200"
+    />
+  </span>
+  <span class="d-badge__label">Chat log</span>
+</span>
```

Comment:
May as well use `<dt-badge>` here.

Reply:
updated in https://github.com/dialpad/dialtone/pull/303/commits/46aafc05dd4f5c990d588d84fd7d6f97ffd80192